### PR TITLE
spec: Move subpackage specific files out of common package

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by okurz** [comment]
https://build.opensuse.org/package/live_build_log/devel:openQA:TestGithub:PR-2887/openQA/openSUSE_Tumbleweed/x86_64 shows

```
[ 1452s] RPM build errors:
[ 1452s]     File not found: /home/abuild/rpmbuild/BUILDROOT/openQA-4.6.1585729138.f40ba16f4-1.1.x86_64/usr/share/openqa/lib/LiveHandler.pm
[ 1452s]     Directory not found: /home/abuild/rpmbuild/BUILDROOT/openQA-4.6.1585729138.f40ba16f4-1.1.x86_64/usr/share/openqa/lib/Parser
[ 1452s]     Directory not found: /home/abuild/rpmbuild/BUILDROOT/openQA-4.6.1585729138.f40ba16f4-1.1.x86_64/usr/share/openqa/lib/Resource
[ 1452s]     Directory not found: /home/abuild/rpmbuild/BUILDROOT/openQA-4.6.1585729138.f40ba16f4-1.1.x86_64/usr/share/openqa/lib/Scheduler
[ 1452s]     File not found: /home/abuild/rpmbuild/BUILDROOT/openQA-4.6.1585729138.f40ba16f4-1.1.x86_64/usr/share/openqa/lib/ServerSideDataTable.pm
[ 1452s]     Directory not found: /home/abuild/rpmbuild/BUILDROOT/openQA-4.6.1585729138.f40ba16f4-1.1.x86_64/usr/share/openqa/lib/Schema
[ 1452s]     Directory not found: /home/abuild/rpmbuild/BUILDROOT/openQA-4.6.1585729138.f40ba16f4-1.1.x86_64/usr/share/openqa/lib/WebAPI
[ 1452s]     Directory not found: /home/abuild/rpmbuild/BUILDROOT/openQA-4.6.1585729138.f40ba16f4-1.1.x86_64/usr/share/openqa/lib/WebSockets
```

**Comment by okurz** [comment]
I will try to fix the flaky coverage in https://github.com/os-autoinst/openQA/pull/2897

**Comment by Martchus** [comment]
There are failed packages which might be related. The failing UI test is likely unrelated.

**Comment by okurz** [comment]
all green now

**Comment by okurz** [comment]
@Martchus the worker is using OpenQA::Setup and that causes the problem in https://github.com/os-autoinst/openQA/blob/207d94566a671b0fa5c3631278f4ea71ec0ec1fa/lib/OpenQA/Setup.pm#L29

**Comment by okurz** [comment]
@os-autoinst/tools-team anyone else sees a problem with this approach or good to go?

**Comment by okurz** [comment]
No problem, this is why I have separate commits. I will just leave out https://github.com/os-autoinst/openQA/pull/2887/commits/f75adbb8ce58fe8c80de4e45a32dee0f15cde8ae . That means that (for now) live handler will be with a special rule be part of the "openQA" main package but not live in WebAPI namespace. If you agree you can approve, I will await successfull checks though to ensure that all packages can be built and also package tests work fine.

File: openQA.spec [hunk 1]
```diff
@@ -410,6 +410,13 @@ fi
 %{_tmpfilesdir}/openqa-webui.conf
 # web libs
 %dir %{_datadir}/openqa
+%{_datadir}/openqa/lib/DBIx/
+%{_datadir}/openqa/lib/LiveHandler.pm
```
Comment (kraih) [inline openQA.spec:414]: That should probably be `%{_datadir}/openqa/lib/OpenQA/LiveHandler.pm`.

Comment (okurz) [reply openQA.spec]: thx, fixed. Also for the intermediate commit about ServerSideDataTable

**Comment by kalikiana** [comment]
> > @Martchus the worker is using OpenQA::Setup and that causes the problem in
> > https://github.com/os-autoinst/openQA/blob/207d94566a671b0fa5c3631278f4ea71ec0ec1fa/lib/OpenQA/Setup.pm#L29
> 
> Ok, then let's ignore it for now. Maybe @kalikiana's PR for splitting the log improves this (because the worker only used `setup_log` from `Setup.pm`).

I think #2912 would be needed for that (that wasn't part of the main log split, which was big enough).

**Comment by okurz** [comment]
only test that failed should be fixed in the future with https://github.com/os-autoinst/openQA/pull/2882 which was done in parallel. I don't see a need to rerun tests here.

**Pull request merged**